kernel/src/fs/ext2/fs.rs

```diff
@@ -200,6 +200,7 @@ impl Ext2 {
         let (_, block_group) = self.block_group_of_ino(ino)?;
         let inode_idx = self.inode_idx(ino);
         block_group.sync_raw_inode(inode_idx, &RawInode::from(inode));
+        // self.sync()
         Ok(())
```

Comment:
The rational sync position is at the end of the `sync_all()`&`sync_data()` in `impl Inode for Ext2Inode {...}`&`impl Inode for ExfatInode {...}` in `kernel/src/fs/ext2/impl_for_vfs/inode.rs`&`kernel/src/fs/exfat/inode.rs`.